gradio/components.py

```diff
@@ -104,14 +104,12 @@ def __init__(
         show_label: bool = True,
         interactive: Optional[bool] = None,
         visible: bool = True,
-        requires_permissions: bool = False,
```

Comment:
Why is this deleted?

Reply:
We used to use `requires_permissions` to figure out if a component needed to request permissions in colab and print a message accordingly. We don't do need to do that anymore. (So unrelated to the original reason I made this PR, but when I saw it, I figured I'd just remove it)